README.md

```diff
@@ -473,6 +473,36 @@ migrations()
 
 ```
 
+## Writing Migrations in Typescript
+
+You can use Typescript to write your migration files using `ts-node`!  First `npm install --save ts-node typescript`,
+then run your migration with ts-node:
+
+```
+node_modules/.bin/ts-node node_modules/.bin/contentful-migration -s $CONTENTFUL_SPACE_ID -a $CONTENTFUL_MANAGEMENT_TOKEN my_migration.ts
```

Comment:
I think `ts-node` is missing here at the beginning of the line, or am I missing something?